charts/apisix-ingress-controller/README.md

```diff
@@ -161,6 +170,7 @@ The same for container level, you need to set:
 | labelsOverride | object | `{}` | Override default labels assigned to Apache APISIX ingress controller resource |
 | nameOverride | string | `""` | Default values for apisix-ingress-controller. This is a YAML-formatted file. Declare variables to be passed into your templates.  |
 | nodeSelector | object | `{}` |  |
+| plugins | list | `[]` | Customize the list of APISIX plugins to enable. By default, APISIX's default plugins are automatically used. See [config-default.yaml](https://github.com/apache/apisix/blob/master/conf/config-default.yaml) |
```

Comment:
Is this auto-generated? I can't find this field in values.yaml

Reply:
my bad i updated helm-docs